README.md

```diff
@@ -47,32 +42,75 @@ cd MyReactApp
 
-### Link
+Include the react native packages in the dependencies of an app's package.json.
+> Note: `@adobe/react-native-aepcore` is required.
 
-[CLI autolink feature](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md) links the module while building the app.
+The following code snippet shows an example of adding libraries in package.json, only include the libraries you need for the app.
+
+```bash
+...
+"dependencies": {
+    "react-native": "0.64.2",
+    "@adobe/react-native-aepcore": "^1.0.0", //includes aepcore, aepsignal, aeplifecycle, aepidentity libraries
+    "@adobe/react-native-aepedge": "^1.0.0", //or replace  with the latest extension version
+    "@adobe/react-native-aepedgeidentity": "^1.0.0", //or replace  with the latest extension version
+    "@adobe/react-native-aepedgeconsent": "^1.0.0", //or replace  with the latest extension version
+    "@adobe/react-native-aepuserprofile": "^1.0.0", //or replace  with the latest extension version
+    "@adobe/react-native-aepassurance": "^3.0.0", //or replace  with the latest extension 
+    "@adobe/react-native-aepmessaging": "^1.0.0-beta.1", //or replace  with the latest extension version
+...
+},
+```
+Inside of the app directory, run
 
-### Initializing
+```bash
+#if using node package manager
+npm install
+```
+or
+```bash
+#if using yarn package manager
+yarn install
+```
 
-Initializing the SDK should be done in native code inside your `AppDelegate` (iOS) and `MainApplication` (Android). The following code snippets demonstrate how to import and register the Mobile Core, Identity, Lifecycle, Signal, and Profile extensions. For other extensions, the documentation on how to initialize the extension can be found in `./packages/{extension}/README.md`
+##### ios development
+For iOS development, after installing the plugins from npm, download the pod dependencies by running the following command:
+```bash
+cd ios && pod install && cd ..
+```
+
+To update native dependencies to latest available versions, run the following command:
+```bash
+cd ios && pod update && cd ..
+```
+## Initializing
+
+Initializing the SDK should be done in native code inside your `AppDelegate` (iOS) and `MainApplication` (Android). The following code snippets demonstrate how to `import` and `register` the Mobile Core, Signal, Lifecycle, Edge Network, EdgeIdentity and EdgeConsent extensions. For documentation on how to initialize each extension can be found in ./packages/{extension}/README.md
 
 ###### **iOS**
 ```objective-c
+//AppDelegate.h
+// must import AEPCore
 @import AEPCore;
 @import AEPServices;
 @import AEPSignal;
-@import AEPLifecycle;
 @import AEPEdge;
 @import AEPEdgeIdentity;
 @import AEPEdgeConsent;
-//@import AEPMessaging;
-//@import AEPUserProfile;
-//@import AEPAssurance;
-//@import AEPIdentity;
+@import AEPLifecycle; 
+//@import AEPMessaging; //enable this extension if you installed @adobe/react-native-aepmessaging
```

Comment:
@cacheung  can we make the same change for the following `@import` statements and extension registration code?